### PR TITLE
Make the number of DynamoDB connections configurable

### PR DESCRIPTION
**Pull request by itaiad200**
It will allow installations to adjust their DDB usage to their loads.

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ This reference uses `.` to denote the nesting of values.
     + **Note:** `endpoint` `aws_region` `aws_access_key_id` `aws_secret_access_key` are not required and used mainly for experimental purposes when working with DynamoDB with different AWS credentials.
       {: .note }
     + `database.dynamodb.health_check_interval` `(duration : 0s)` - Interval to run health check for the DynamoDB instance (won't run if equal to 0).
+    + `database.dynamodb.max_attempts` `(int : 10)` - The maximum number of attempts to perform on a DynamoDB request
+    + `database.dynamodb.max_connections` `(int : 0)` - The maximum number of connections to DynamoDB. 0 means no limit.
```
Comment (guy-har) [inline docs/reference/configuration.md:61]: Are we sure we want to change the default to unlimited?

**Review by guy-har — APPROVED** [review]
LGTM

File: docs/reference/configuration.md (hunk 1 above)
Comment (itaiad200) [reply docs/reference/configuration.md:61]: Yes, I don't think we should have a default limitation on one of the system's bottlenecks: the kv store.

Comment (itaiad200) [reply docs/reference/configuration.md:61]: Unless I'm missing something

**Pull request merged**